src/main/scala/io/moia/protos/teleproto/PbResult.scala

```diff
@@ -116,6 +120,8 @@ final case class PbFailure(errors: Seq[(String, String)]) extends PbResult[Nothi
   def toTry: Try[Nothing] = Failure(new Exception(toString))
 
   def toOption: Option[Nothing] = None
+
+  override def toEither: Left[Seq[(String, String)], Nothing] = Left(errors)
```

Comment:
This is very similar to `toTry` and `toOption` functions above. This one has an `override` modifier but the others don't. Please make it consistent by adding it to the others. Or remove it from here but I'd prefer the former.